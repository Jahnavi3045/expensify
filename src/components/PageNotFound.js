import React from 'react'
import { Link } from 'react-router-dom'

const PageNotFound = () => {
  return (
    <div>
      404-PAGE NOT FOUND
      <Link to='/'>Go to Home page</Link>
    </div>
  )
}

export default PageNotFound
