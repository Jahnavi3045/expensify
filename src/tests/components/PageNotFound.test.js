import React from "react";
import ReactShallowRenderer from 'react-test-renderer/shallow'
import PageNotFound from "../../components/PageNotFound";

test('should render 404 page correctly',()=>{
    const renderer=new ReactShallowRenderer()
    renderer.render(<PageNotFound/>)
    expect(renderer.getRenderOutput()).toMatchSnapshot()
})