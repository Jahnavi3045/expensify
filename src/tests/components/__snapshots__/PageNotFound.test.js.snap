// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 404 page correctly 1`] = `
<div>
  404-PAGE NOT FOUND
  <Link
    to="/"
  >
    Go to Home page
  </Link>
</div>
`;
